core/classes/Core/Email.php

```diff
@@ -10,25 +10,48 @@
 */
 
 use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
 
 class Email {
 
+    public const REGISTRATION = 1;
```

Comment:
Could do with making this extensible (at some point, I appreciate it wasn't before!) by other modules (FORUM_TOPIC_REPLY for example is obviously not part of the core, rather the Forum module)